src/helper-modules/govuk-notify/dto/post-emails-response.dto.ts

```diff
@@ -24,6 +24,7 @@ export class PostEmailsResponseDataContent {
     example: null,
     description: 'Unsubscribe links are not used, so this field should always be null',
     nullable: true,
+    required: false,
```

Comment:
It's strange there are no tests that require updating. Not a blocker.

Reply:
Agree some tests are in order will cover in seprate PR.